server/views/pages/bookedtoday/arrivals/searchforexisting/search/changeDateOfBirth.njk

```diff
@@ -0,0 +1,54 @@
+{% extends "../../../../../partials/layout.njk" %}
```

Comment:
split out (duplicated) these as discussed so have different versions between searching and changing details